packages/core/src/index.html

```diff
@@ -243,7 +243,7 @@
         <meta name="google-site-verification" content="g5u3TyEdmYJ_HKZ6EmpW1nlKwD4XPVeLMhN9leeMfuw" />
         <meta property="og:title" content="Deriv" />
         <meta property="og:type" content="website" />
-        <meta property="og:image" content="/images/common/og_image.gif" />
+        <meta property="og:image" content="" />
```

Comment:
should we add the same comment here?

Reply:
Thanks for noticing this. added a comment.